src/errors.exceptions.md

```diff
@@ -4,8 +4,8 @@ In general, prefer [explicit error handling mechanisms](errors.result.md).
 
 Annotate each module at top-level (before imports):
 
-* `{.push raises: [Defect].}` (nim 1.2)
-* `{.push raises: [].}` (nim 1.4+)
+- `{.push raises: [Defect].}` (nim 1.2)
```

Comment:
what's wrong with `*`?

Reply:
Sorry, it's just an autoformatter. I can revert these if you prefer `*`